src/core.js

```diff
@@ -272,6 +272,16 @@ $.extend( $.validator, {
 				this.element( element );
 			}
 		},
+		oninput: function( element ) {
+			if ( element.name in this.submitted || element.name in this.invalid ) {
+				this.element( element );
+			} else {
+				return;
+			}
+		},
+
+		// DEPRECATED
+		// Please use `oninput` instead.
 		onkeyup: function( element, event ) {
```

Comment:
Would it be ok to remove one of the 2 handlers if one of the events is supported.
So:
* if `input` event is supported, we can then delete `onkeyup`
* else, remove `oninput`.

So the user now is supposed to use the 2 handlers to support legacy browsers (even though I think it's a waste of time trying to support <IE9 and other legacy browsers)